SPECS-EXTENDED/supermin/supermin.spec

```diff
@@ -30,25 +25,15 @@ Distribution:   Mariner
 
 Summary:       Tool for creating supermin appliances
 Name:          supermin
-Version:       5.2.0
-Release:       3%{?dist}
+Version:       5.2.1
+Release:       1%{?dist}
 License:       GPLv2+
-
-%if 0%{?rhel} >= 7
-ExclusiveArch: x86_64
-%endif
-
-ExcludeArch:   %{ix86}
-
-URL:           http://people.redhat.com/~rjones/supermin/
-Source0:       http://download.libguestfs.org/supermin/%{source_directory}/%{name}-%{version}.tar.gz
-Source1:       http://download.libguestfs.org/supermin/%{source_directory}/%{name}-%{version}.tar.gz.sig
+URL:           https://github.com/libguestfs/supermin
+Source0:       https://download.libguestfs.org/supermin/%{source_directory}/%{name}-%{version}.tar.gz
+Source1:       https://download.libguestfs.org/supermin/%{source_directory}/%{name}-%{version}.tar.gz.sig
```

Comment:
Could we remove it along with `Source2` and the GPG verification? We're using signatures for our sources anyway.

Reply:
Will fix, thanks

Reply:
The `*.sig` file is still here along with the files themselves and the signatures.

Reply:
Removed